qemu/tests/x86_cpu_model_boot.py

```diff
@@ -0,0 +1,149 @@
+import os
+import uuid
+import logging
+
+from avocado.utils import path
+from virttest import cpu
+from virttest import virsh
+from virttest import data_dir
+from virttest import libvirt_xml
+from virttest import env_process
+from virttest import error_context
+
+
+def get_flags_patterns(flags_dict):
+    """
+    Return all possible patterns for given flags
+    :param flags_dict: The original dict of flags
+    """
+    tmp_dict = {}
+    replace_char = [('_', ''), ('_', '-'), ('-', '_'), ('-', '')]
+    for flag in flags_dict.keys():
+        tmp_list = []
+        tmp_list.extend(list(map(lambda x: flag.replace(*x), replace_char)))
+        for tmp in tuple(tmp_list):
+            if tmp != flag:
```

Comment:
I guess even when tmp==flag, we can still  assign tmp_dict[tmp], then return tmp_dict directly.

Reply:
> I guess even when tmp==flag, we can still assign tmp_dict[tmp], then return tmp_dict directly.

update and thanks.